### DISCUSSION TITLE
Enable "authenticator" 20-slot bag (+4 slots)

### DISCUSSION THREAD

**Discussion by ChrisFredriksson** [azerothcore/azerothcore-wotlk#17228] [General]
Is there a way to enable the "authenticator" +4 slots on the original bag? So it becomes 20-slot.

I've searched for a couple of weeks, but if there is a way I just can't have the right search words for it.

Adding 2FA to the account did not enable the +4 slots on the bags.

If this isn't possible - that is fine. I have seen other servers in the past have this enabled, but that could of course be their own coding that they haven't shared with the community.

Thanks in advance!

**Comment by Nyeriah** [comment]
Unfortunately this is a legion client change therefore doesn’t exist in the client we use, thus we cannot enable it

**Comment by ChrisFredriksson** [comment]
So strange, I thought this came much earlier.. Thanks!
